### ISSUE TITLE
ENV directory is not calculated correctly

### ISSUE THREAD

**Issue by manuviswam** [getgauge/gauge#1278]
When gauge project is using gradle and the gauge project is under a subdirectory, gauge is unable to find environments.

A sample project has been created https://github.com/manuviswam/gauge-failure-demo

In the above project if you run `./gradlew clean gauge -Penv=foo --info` from the root directory, test will fail because it is unable to find the environment `foo`

If you run a similar comman `../gradlew clean gauge -Penv=foo --info` from the `gauge` directory under the root folder, this test will pass.

This does not seem to be a gradle plugin issue since when I run the command `gauge run --env foo <path_to_root_folder>/gauge specs` it is still not able to find the environment

### Expected behavior
Environment should be picked up from the right location

### Actual behavior
Environment not found

### Steps to reproduce
1. Checkout this repo https://github.com/manuviswam/gauge-failure-demo
2. Run `./gradlew clean gauge -Penv=foo --info`

### Gauge version
```
Gauge version: 1.0.3

Plugins
-------
```
html-report (4.0.6)
java (0.6.9)
js (2.3.2)
screenshot (0.0.1)
```

**Comment by sguptatw** [comment]
Verified with version
```
Gauge version: 1.0.4.nightly-2018-12-25
Commit Hash: fe5e7da

Plugins
-------
java (0.7.0.nightly-2018-12-28)
```

**Comment by manuviswam** [comment]
When will this fix available in release?

**Comment by Apoorva-GA** [comment]
Fix is available in [Gauge 1.0.4](https://github.com/getgauge/gauge/releases/tag/v1.0.4)
